pywbem/cim_obj.py

```diff
@@ -3659,28 +3675,32 @@ class CIMQualifierDeclaration(_CIMComparisonMixin):
         `None` means that this information is not available.
 
       toinstance (:class:`py:bool`):
-        If not `None`, specifies whether the qualifier value propagates
-        to instances.
+        If not `None`, specifies the flavor that defines whether the qualifier
+        value propagates to instances.
```

Comment:
I don't like the new wording: "specifies the flavor that defines whether" is an indirection that does not help me to understand it.

We may need some change to better express that it only represents what the qualifier declaration specified. However, that would be a change on the sentence describing None, e.g.:

`None` means that the qualifier declaration does not specify this information.

And that change would need to be done on all flavor parameters.